week02/teach/ComplexStackSolution.cs

```diff
@@ -1,13 +1,11 @@
-﻿namespace teach_03; 
-
-public static class ComplexStackSolution {
+﻿public static class ComplexStackSolution {
     public static void Main() {
-        // True (passes on line 46 ... stack was empty at the end)
+        // True (passes on line 33 ... stack was empty at the end)
```

Comment:
I'm nervous about calling out line numbers explicitly here. it seems like it will be difficult to keep everything in sync and maintained. I wonder if there is a better way to reference the code here.

Reply:
Fixed by removing the line numbers. I think the comment in parenthesis after the line number should be sufficient to  explain why the code behaved as it did.